packages/axios-extension/src/abap/adt/handlers/transport.ts

```diff
@@ -0,0 +1,97 @@
+import XmlParser from 'fast-xml-parser';
+import * as xpath from 'xpath';
+import { DOMParser } from '@xmldom/xmldom';
+
+export function parseTransportRequests(xml: string): string[] {
```

Comment:
Find a better name than `parseTransportRequests`? Parsing technically means to break something down into components. What are we doing exactly? Extracting transport request numbers from the response? Some JS Doc please?

Reply:
The function name has been updated. Not sure why it is not updated here. Added JS doc